nbs/models.ipynb

```diff
@@ -83,6 +83,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.                    config['n_series'] = n_series
</pre><p>I think this would work as well if you set it before the <code>if backend == 'optuna':</code> and then we can get rid of the <code>if backend == 'ray':</code>below. Same for the other ones</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>

Reply:
Ah I see, I thought we were only setting it for the default config. We should probably remove it from the auto model then so that the user always provides it through the config (and an error will be raised if it's not provided). WDYT?

Reply:
I came up with a method of checking the supplied config for `'n_series' = n_series` in case of Optuna backend, by using the MockTrial you created earlier. It raises an Exception if the user does not supply  `'n_series' = n_series`.

Reply:
Sorry, I missed your last comment (should have renewed the page). Yes, I can change it so that it always throws an Exception (now it only throws an Exception when using Optuna).

wdyt?

Reply:
Removing it from the init signature is a breaking change, but I think that's the better option for the future. We can deprecate it and issue a FutureWarning indicating that the n_series should be provided through the config if it's provided in the init signature for the multivariate auto models.

Reply:
@jmoralez In general, why do we ever want the user to supply `n_series` in MultiVariate models? We can infer it ourselves from `unique_id` in the dataset, right? In what situation is it not true that `n_series == unique(unique_id)`?

(This is more in the sense that if we make a breaking change to remove it, why not remove it entirely across all MultiVariate models)

Reply:
Fiddled around with it a bit but think it's indeed better to remove `n_series ` everywhere (i.e., also from the base multivariate models) rather than only from Auto* models, if we want to make that breaking change. 

wdyt?

Reply:
Ok, then I suggest we leave it as it is now (also for Auto*)

Reply:
Agree